DESCRIPTION

```diff
@@ -18,7 +18,9 @@ Imports:
     R6 (>= 2.0.0),
     stringi (>= 0.3.0),
     jsonlite (>= 0.9.16),
-    httpuv (>= 1.2.3)
+    httpuv (>= 1.2.3),
+    feather,
```

Comment:
Absolutely! I had not looked carefully enough at how you are handling this. Your solution is elegant. Also, perhaps I should focus on a more generic solution to POST binary files in plumber and leave the handling of feather or other formats to the API code.